.pre-commit-config.yaml

```diff
@@ -10,11 +10,19 @@ repos:
   - repo: https://github.com/pre-commit/pre-commit-hooks
     rev: v4.6.0
     hooks:
-      - id: trailing-whitespace
       - id: check-case-conflict
+      - id: check-executables-have-shebangs
+      - id: check-json
+      - id: check-shebang-scripts-are-executable
+      - id: check-merge-conflict
+      - id: check-symlinks
+      - id: destroyed-symlinks
```

Comment:
we don't have symlinks because Windows won't support them...

Reply:
This hooks aims to detect them. But since we dont have any symlink I think i can remove

Reply:
> we don't have symlinks because Windows won't support them...

Fun fact: Windows versions since Vista do have symbolic link support. It is just that they are generally not that widely used as creating them required admin elevation and creating them is hard and obscure.

Reply:
Yeah, I guess we can just remove this check and save a few seconds in CI 🙂

Reply:
Removed with 8046a5f